package.json

```diff
@@ -4,7 +4,6 @@
   "description": "An HTTP/1.1 client, written from scratch for Node.js",
   "main": "index.js",
   "types": "index.d.ts",
```

Comment:
Not sure if it's needed, but if we add a `"exports": "./index.js"` we can use self-resolution too.

Reply:
I recommend against using `exports` in any way.